src/lib/classes/notification.js

```diff
@@ -461,7 +461,7 @@ export class Notification {
          * Validate the list of notification IRIs and return a list of valid ones
          */
         const coreNotificationShape =
-          'https://shexshapes.inrupt.net/public/notifications/app-notification.shex';
+          'https://shexshapes.inrupt.net/public/notifications/core-notification.shex';
```

Comment:
I recall we had a discussion about this in a previous PR, but I don't remember the conclusion we reached. Should this change be kept or reverted ?

Reply:
Yeah, I reverted this change after chatting with Kevin.